system/utils.py

```diff
@@ -136,6 +136,9 @@ def random_seed_and_json():
                                  "remaining..., will sleep for {}".format(fname, remain, sleep_dur))
                 await asyncio.sleep(sleep_dur)
             else:
+                print("\n{} failed; not trying any more because {} "
+                      "seconds have passed; args were {}".
+                      format(fname, timeout, args))
```

Comment:
why do you think you need this? `pytest` provides ability to pass logs to stdout using `--log-cli-level` option

Reply:
Removed.